db/drizzle.ts

```diff
@@ -0,0 +1,13 @@
+import postgres from "postgres";
+import { drizzle } from "drizzle-orm/postgres-js";
+import "dotenv/config";
```

Comment:
No need to install dotenv, you can just import '../envConfig' that uses '@next/env'